src/client/Microsoft.Identity.Client/AppConfig/PopAuthenticationConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using Microsoft.Identity.Client.AuthScheme.PoP;
+
+namespace Microsoft.Identity.Client.AppConfig
+{
+    /// <summary>
+    /// Configuration properties used to construct a Proof of possesion request
+    /// </summary>
+    public class PopAuthenticationConfiguration
+    {
+        /// <summary>
+        /// An HTTP uri to the protected resource which requires a PoP token. The PoP token will be cryptographically bound to the request.
+        /// </summary>
+        public Uri RequestUri { get; set; }
+
+        /// <summary>
+        /// The HttP method to be bound to the request
+        /// </summary>
+        public HttpMethod PopHttpMethod { get; set; }
+
+        /// <summary>
+        /// A provider that can handle the asymmetric key operations needed by POP, that encapsulates a pair of public and private keys
+        /// and some typical crypto operations
+        /// </summary>
+        public IPoPCryptoProvider PopCryptoProvider { get; set; }
```

Comment:
I wonder if we should actually put this crypto provider at the Application level instead of the Request level ? This way we can cache things like Thumbprint. 

I suggest that for now we go with this and we discuss this in the API Review.